Dockerfile

```diff
@@ -85,7 +85,8 @@ COPY . .
 # create the rbac user
 RUN \
     adduser rbac -u ${USER_ID} -g 0 && \
-    chmod ug+rw ${APP_ROOT} ${APP_HOME} ${APP_HOME}/static /tmp
+    chmod ug+rw ${APP_ROOT} ${APP_HOME} ${APP_HOME}/static && \
```

Comment:
Changing the permissions to prometheus /tmp folder in order to provide the correct permissions for the prometheus metrics within cache.py.

Fixing the issue about the permission issues that are arising whenever ephemeral environment is being run. After talking with Jacob, we believe that this should be okay security wise since the folder is a temporary folder.